### PR TITLE
Readme usage section

### PR DESCRIPTION
**Pull request by maiqueb**

### CODE REVIEW

**Comment by maiqueb** [comment]
@jcaamano Please review :)

File: README.md [hunk 1]
```diff
@@ -11,11 +11,34 @@ We encourage prospective users to try out the demo, and use the provided
 [manifests](two_macvtap_fedora_vms.yaml) as reference for more advanced
 use cases.
 
+# Installation
+To be able to run the demo, you'll need to clone the following repos:
+  - [kubevirt](https://github.com/kubevirt/kubevirt/):
+    `git clone https://github.com/maiqueb/kubevirt/ -branch add_macvtap_binding_mech`
```
Comment (maiqueb) [inline README.md:17]: would it be better to fetch the PR to the original kubevirt repo instead ?

Comment (jcaamano) [reply README.md:17]: No, I think is fine as it is. It's hopefully temporary anyway.

File: README.md [hunk 2]
```diff
@@ -11,11 +11,34 @@ We encourage prospective users to try out the demo, and use the provided
...
+```bash
+# start a kubernetes cluster w/ kubevirt / multus / macvtap device plugin
+# already configured.
+# It defaults to 2 nodes, and the k8s-multus-1.13.3 provider.
+KUBEVIRT_REPO=<kubevirt_repo_location> \
+    MACVTAP_PLUGIN_REPO=<macvtap_device_plugin_repo_location> \
+    ./start_cluster.sh
+```
+
+Once your k8s cluster is started, you can then , be executing:
```
Comment (jcaamano) [inline README.md]: > you can then ... ?

Comment (maiqueb) [reply README.md]: Done.

**Review by jcaamano — APPROVED** [review]

**Pull request merged**